Python/codegen.c

```diff
@@ -1855,6 +1857,7 @@ codegen_jump_if(compiler *c, location loc,
     VISIT(c, expr, e);
     ADDOP(c, LOC(e), TO_BOOL);
     ADDOP_JUMP(c, LOC(e), cond ? POP_JUMP_IF_TRUE : POP_JUMP_IF_FALSE, next);
+    ADDOP(c, NO_LOCATION, NOT_TAKEN);
```

Comment:
Could `ADDOP_JUMP` just add the `NOT_TAKEN` when the opcode is a conditional jump?

Reply:
I've done that and it cleans things up a bit.